apps/wso2-digital-wallet/src/pages/Home/Home.css

```
@@ -352,36 +369,68 @@
   font-weight: 600;
   color: #1C1917;
   letter-spacing: -0.3px;
-  margin-bottom: 1px;
+  line-height: 1.2;
 }
 
-.recent-activity-address {
+.transaction-item-counterparty {
   font-size: 11px;
   color: #9CA3AF;
   font-family: 'JetBrains Mono', 'Monaco', 'Menlo', 'Ubuntu Mono', monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)